tests/queries/0_stateless/01049_join_low_card_bug_long.sql.j2

```diff
@@ -98,13 +100,13 @@ SET join_use_nulls = 0;
 
 SELECT lc, toTypeName(lc)  FROM l_lc AS l RIGHT JOIN r_lc AS r USING (x) ORDER BY l.lc;
 
-SELECT lowCardinalityKeys(lc.lc) FROM r FULL JOIN l_lc as lc USING (lc) ORDER BY lowCardinalityKeys(lc.lc);
```

Comment:
In new analyzer `lc.lc` became String (because r contains `String` and we take supertype as a result type), so lowCardinalityKeys is not applicable. We can leave this query only for old analyzer, but I don't think it's essential.